app/models/nta_upload.rb

```diff
@@ -273,6 +300,244 @@ def NtaUpload.load(id, series_id)
     du.full_load
   end
 
+  def NtaUpload.average(id, group)
+    # this method is a noop/placeholder
+    "#{id} #{group}"
+  end
+
+  def delete_universe(universe)
```

Comment:
This seems dangerous since 'UHERO' could be passed in and torch everything. Since 'NTA' is the only universe we want to delete in this way, and since this lives in the NtaUpload model file, maybe we should make this only delete the 'NTA' universe.